Clean up grammar in two brother's 01

jrmiller82 opened the pull request:
I cleaned up some grammer to make it read more naturally. There is so
much passive voice and weak verbs in the original. However, I've tried
to retain the same events; I just made it read a bit better. I'm also
open to any suggestions you may have; just wanting to make Wesnoth the
best it is.

beetlenaut commented:
I don't agree with this change. You have changed a lot of archaic forms to 
more modern ones, but the archaic forms were more appropriate for the genre. 
You're right that it reads more like natural, modern English, but that's not a 
good thing in this case.

I don't know for sure what you mean by too much passive voice, because the 
only passive voice I see is in two phrases that you added! Maybe you mean the 
perfect continuous forms, which you changed a lot of. However, I think they 
were better because they gave a sense of immediacy to the events.

---Dan Gerhards

On Saturday, November 28, 2015 4:45:24 PM James Miller wrote:
> I cleaned up some grammer to make it read more naturally. There is so
> much passive voice and weak verbs in the original. However, I've tried
> to retain the same events; I just made it read a bit better. I'm also
> open to any suggestions you may have; just wanting to make Wesnoth the
> best it is.
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/wesnoth/wesnoth/pull/556
> 
> -- Commit Summary --
> 
>   * Clean up grammar in two brother's 01
> 
> -- File Changes --
> 
>     M data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg (20)
> 
> -- Patch Links --
> 
> https://github.com/wesnoth/wesnoth/pull/556.patch
> https://github.com/wesnoth/wesnoth/pull/556.diff
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/wesnoth/wesnoth/pull/556

jrmiller82 commented:
I removed the pull request after discussing on #wesnoth-dev. I think I just can't stand the deliberate archaisms. Feel free to use what you like or discard it. 

Sent from my iPhone

> On Nov 29, 2015, at 22:04, beetlenaut <notifications@github.com> wrote:
> 
> I don't agree with this change. You have changed a lot of archaic forms to 
> more modern ones, but the archaic forms were more appropriate for the genre. 
> You're right that it reads more like natural, modern English, but that's not a 
> good thing in this case.
> 
> I don't know for sure what you mean by too much passive voice, because the 
> only passive voice I see is in two phrases that you added! Maybe you mean the 
> perfect continuous forms, which you changed a lot of. However, I think they 
> were better because they gave a sense of immediacy to the events.
> 
> ---Dan Gerhards
> 
> On Saturday, November 28, 2015 4:45:24 PM James Miller wrote:
> > I cleaned up some grammer to make it read more naturally. There is so
> > much passive voice and weak verbs in the original. However, I've tried
> > to retain the same events; I just made it read a bit better. I'm also
> > open to any suggestions you may have; just wanting to make Wesnoth the
> > best it is.
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/wesnoth/wesnoth/pull/556
> > 
> > -- Commit Summary --
> > 
> > * Clean up grammar in two brother's 01
> > 
> > -- File Changes --
> > 
> > M data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg (20)
> > 
> > -- Patch Links --
> > 
> > https://github.com/wesnoth/wesnoth/pull/556.patch
> > https://github.com/wesnoth/wesnoth/pull/556.diff
> > 
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/wesnoth/wesnoth/pull/556
> 
> —
> Reply to this email directly or view it on GitHub.
>

CelticMinstrel commented:
The changes on lines 235, 314, and 319 may be worth considering either partially or fully. Some of those lines definitely need work, even if it's not in the direction this PR takes it.
